test/test_ops.py

```diff
@@ -655,6 +656,7 @@ def script_func_no_mask(x_, offset_, weight_, bias_, stride_, pad_, dilation_):
 
     @unittest.skipIf(not torch.cuda.is_available(), "CUDA unavailable")
     def test_autocast(self):
+        set_rng_seed(0)
```

Comment:
The DeformConv2d autocast method is flaky and the different random seed causes it to fail. By fixing it here we avoid breaking master. It would be good to investigate whether the test can be changed to make it less flaky. From what I see it's not a problem of tolerance since we used the standard reduced value.

Reply:
Might be good to put a note in the code or in an issue to track that this is flaky. Can you open an issue tracking this?

Reply:
Issue #3060.